gammapy/scripts/downloadclasses.py

```diff
@@ -117,17 +117,21 @@ def getfilelist(self):
             if self.modetutorials and not self.listfiles:
                 sys.exit()
 
-            url = DEV_DATA_JSON_URL
             if self.release:
                 filename_datasets = "gammapy-" + self.release + "-data-index.json"
                 url = BASE_URL + "/data/" + filename_datasets
-
-            log.info("Reading {}".format(url))
-            try:
-                txt = urlopen(url).read().decode("utf-8")
-            except Exception as ex:
-                log.error(ex)
-                return False
+                log.info("Reading {}".format(url))
+                try:
+                    txt = urlopen(url).read().decode("utf-8")
+                except Exception as ex:
+                    log.error(ex)
+                    return False
+            else:
+                # for development just use the local index file
+                filename = (Path(__file__).parent / DEV_DATA_JSON_LOCAL).resolve()
+                log.info("Reading {}".format(filename))
+                with filename.open(mode="r") as f:
```

Comment:
@adonath - I've started to use this coding pattern:
```
txt = Path(...).read_text()
```
I'm 99% sure this does close the file as well, I think I checked that. It's usually shorter both in terms of lines and in terms of indentation (no with statement needed).